smoke-tests/designer/features/pageobjects/sections/createComponent.section.js

```diff
@@ -1,10 +1,14 @@
-const Page = require("./basePage");
+const Page = require("../pages/basePage");
 
-class AddComponentPage extends Page {
+class CreateComponentSection extends Page {
```

Comment:
are you actually using the methods in the parent class `Page`?

Reply:
@jenbutongit I am but I plan to refactor it out and it's leading to traceability issues

Reply:
Let's pair (whenever I get time...). IIRC most of your classes don't need a constructor, they're mostly just convenience methods which might be causing some issues. It might also be that some of the exports are singletons (init-ed once, but exported everywhere) which might be confusing the stack trace